inference-engine/include/ie_core.hpp

```diff
@@ -70,15 +71,18 @@ class INFERENCE_ENGINE_API_CLASS(Core) {
      * For IR format (*.bin):
      *  * if path is empty, will try to read bin file with the same name as xml and
      *  * if bin file with the same name was not found, will load IR without weights.
-     * ONNX models with data files are not supported
+     * For ONNX format (*.onnx or *.prototxt):
+     *  * binPath parameter is not used.
      * @return CNNNetwork
      */
     CNNNetwork ReadNetwork(const std::string& modelPath, const std::string& binPath = {}) const;
     /**
      * @brief Reads models from IR and ONNX formats
      * @param model string with model in IR or ONNX format
      * @param weights shared pointer to constant blob with weights
-     * ONNX models doesn't support models with data blobs.
+     * Reading ONNX models doesn't support loading weights from data blobs.
+     * If you are using an ONNX model with external data files, please use the
+     * ReadNetwork function overload which takes a filesystem path to the model.
```

Comment:
can we refer to exact overloading? E.g. `use InferenceEngine::Core::ReadNetwork(const std::string& model, const Blob::CPtr& weights) const`. you can build docs via make openvino_docs and see how it looks like

Reply:
@ilya-lavrenov done, the proper link was generated.
![image](https://user-images.githubusercontent.com/11331953/95579308-ac962e80-0a35-11eb-9ac6-b6e5d584c5fe.png)